LoopTests/Models/RemoteCommandTests.swift

```diff
@@ -0,0 +1,107 @@
+//
+//  RemoteCommandTests.swift
+//  LoopTests
+//
+//  Created by Bill Gestrich on 8/13/22.
+//  Copyright © 2022 LoopKit Authors. All rights reserved.
+//
+
+import XCTest
+import HealthKit
+@testable import Loop
+
+class RemoteCommandTests: XCTestCase {
+
+    override func setUpWithError() throws {
+    }
+
+    override func tearDownWithError() throws {
+    }
+    
+    
```

Comment:
Added RemoteCommand tests, just for the feature I am changing - Carb entries. This could be a good place to add the other tests as we make more changes.